src/opt/serve.rs

```diff
@@ -2,7 +2,8 @@ use anyhow::Context;
 use clap::Parser;
 use directories::ProjectDirs;
 use futures::{stream::FuturesUnordered, StreamExt, TryStreamExt};
-use penumbra_crypto::{Value, Zero};
+use num_traits::identities::Zero;
```

Comment:
This `num_traits` import was recommended by the compiler, to satisfy the `is_zero()` call here: https://github.com/penumbra-zone/galileo/blob/39fd7cda84c1034b85967624b9ca2a9d2c3d60e9/src/opt/serve.rs#L69 Not sure this is functionally equivalent to the code we had prior to the refactor.